bundles/binding/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/HueGenericBindingProvider.java

```diff
@@ -84,10 +84,12 @@ public void processBindingConfiguration(String context, Item item,
 
 		super.processBindingConfiguration(context, item, bindingConfig);
 
-		try {
-
-			if (bindingConfig != null) {
+		if( bindingConfig == null || bindingConfig.length() < 1 ) {
+				logger.warn("No settings configured for Hue binding for item '{}'.", item);
```

Comment:
Ok. I think everything should be good to go.

I ran a format from within the IDE, pushed the changes and squashed all the recent commits. Any other cleanup needed?